spec.md

```diff
@@ -256,6 +258,26 @@ Broker's catalog, the Platform might not have an end-user with which to
 associate the request, therefore in those cases the originating identity header
 would not be included in those messages.
 
+### Request Identity
+
+A platform might wish to uniquely identify a specific request as it flows throughout the system.
+For example, this might be used for logging for request tracing purposes. In order to facilitate
+this, platforms will need to provide identifacation information to the Service Broker for each
+request. Platforms MAY support this feature, and if they do, they MUST adhere to the following:
```

Comment:
This is a good explanation on why request identity is needed, thank you!

Reply:
s/platform/Platform